util/float.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package util
+
+import (
+	"math"
+)
+
+// Round returns the number with the specified precision.
+func Round(number float64, precision int) float64 {
```

Comment:
I'd prefer if we make this an unexported function in the cover package. We've been trying to reduce the export surface area in OPA for a while now.